lib/assets/javascripts/twine.coffee

```diff
@@ -56,6 +56,7 @@
 
   bind = (context, node, indexes, forceSaveContext) ->
     currentBindingCallbacks = []
+    element = null
```

Comment:
I like being explicit about scope, otherwise it seems like a side-effect that the variable is being set at all